operatorconfig/driverconfig/powerflex/v2.13.1/node.yaml

```diff
@@ -275,7 +275,7 @@ spec:
           hostPath:
             path: <KUBELET_CONFIG_DIR>/pods
             type: Directory
-         # will be removed if installing on OpenShift
+        # will be removed if installing on OpenShift
```

Comment:
What will be removed, the following stanza or the contents (mount) of the stanza?

Reply:
the volume and mount below,
scaleio-path-bin
due to the issue we saw when installing on OCP

Reply:
I was asking about who/what/when. You answered the what. But who is supposed to remove the item. Is this automatic or require manual intervention. Is it done during rendering of the template?